regolith/schemas.py

```diff
@@ -7,7 +7,8 @@
 from .sorters import POSITION_LEVELS
 
 SORTED_POSITION = sorted(POSITION_LEVELS.keys(), key=POSITION_LEVELS.get)
-
+PRESENTATIONS_TYPE = ["award", "colloquium", "contributed_oral", "invited", "keynote",
```

Comment:
this is now not correct.   webinar is not a valid type in presentations.

Reply:
Done!